lua/autorun/server/sv_increase_starting_ammo.lua

```diff
@@ -1,22 +1,17 @@
+BlacklistedAmmo = { 
+    ["Smoke Grenades"] = true,
+    ["Flash Grenades"] = true,
+    ["Frag Grenades"] = true,
+    ["40MM"] = true
+    -- we don't actually have to add RPG ammo here
+}
 
-local function givePlayerAmmoForWeapon( player, weapon )
-    local ammoType = weapon:GetPrimaryAmmoType()
-    local currentAmmo = player:GetAmmoCount( ammoType )
-    if ( currentAmmo == 0 ) then
-        player:GiveAmmo( 90, ammoType, false )
-    end
-end
-
-local function giveCW2AmmoOnSpawn( player )
-    timer.Simple( 0, function()
-        for _, weapon in pairs( player:GetWeapons() ) do
-            local isCW2Weapon = string.sub( weapon:GetClass(), 1, 2 ) == "cw"
-            if ( isCW2Weapon ) then
-                givePlayerAmmoForWeapon( player, weapon )
-            end
+function givePlayerCWAmmo(Ply)
+    for AmmoName, bulletInfo in pairs(CustomizableWeaponry.ammoTypes) do
+        if not BlacklistedAmmo[AmmoName] then
+            Ply:SetAmmo(1000,AmmoName)
```

Comment:
```suggestion
            Ply:SetAmmo( 1000, AmmoName )
```
space inside parentheses and especially after commas